torch/export/_trace.py

```diff
@@ -1260,6 +1278,7 @@ def forward(self, *args, **kwargs):
         if free_unbacked_symbols(k)
     }
     gm.meta["forward_arg_names"] = forward_arg_names
+    gm.meta["disable_forced_specializations"] = _disable_forced_specializations
```

Comment:
Where's this used? Also this is strict, yeah? So redundant with your current assumption that strict is excluded.